_targets.R

```diff
@@ -6,10 +6,40 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+nwis_Cd <- '00010'
```

Comment:
I like that you defined the parameter code here in the `_targets.R` file. Could you also specify the start date and end date here, in case anyone wanted to change those without digging into the `download_nwis_site_data()` function?

Reply:
Sounds good, done!